packages/jsii-calc/package.json

```diff
@@ -25,7 +25,7 @@
   "main": "lib/index.js",
   "types": "lib/index.d.ts",
   "scripts": {
-    "build": "jsii && jsii-rosetta --compile",
+    "build": "jsii --disable-reserved-words-warnings && jsii-rosetta --compile",
```

Comment:
This would be more idiomatic if it was `--no-reserved-words-warnings`. I'm also afraid that this is not going to scale super well in the future.

Maybe it'd be smoother if we had a `--silence-warnings=<comma-separated-list-of-types>`, e.g: `--silence-warnings=reserved-word`.

Reply:
I don't want to over engineer this. I will let you do that as part of #1073. This is a temporary quick fix.

Reply:
As soon as it's part of the CLI signature, it's forever... So I don't care so much if you get the "over engineered" argument and then internally map it to the boolean signal, but I don't want to bring in new CLI arguments that we'll throw away later.

Reply:
we can always deprecate (and also reminding you that jsii is pre 1.0)

Reply:
But ok, I'll change this `--silence-warnings` if you feel strongly about it

Reply:
Please :)